src/logcollector/logcollector.c

```diff
@@ -152,6 +152,17 @@ void LogCollectorStart()
                        ARGV0);
             }
         }
+        else if (strcmp(logff[i].logformat, "journald") == 0) {
+#ifdef HAVE_SYSTEMD
+            verbose(READING_JOURNAL, ARGV0, logff[i].file);
```

Comment:
`READING_JOURNAL` needs to be declared somewhere (`src/error_messages/error_messages.h`?)